app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -896,12 +903,14 @@ class PipelineSampleReport extends React.Component {
           aria-hidden="true"
         />
       );
-    if (reportDetails.assembled_taxids.indexOf(taxInfo.tax_id.toString()) >= 0)
-      assemblyDot = (
+    if (taxInfo.tax_id > 0 && taxInfo.tax_level == 1 && taxInfo.NT.r > 0)
```

Comment:
As discussed, constrain to admin for now.

Reply:
done